numerics_paper/version2/numerical_chj.tex

```diff
@@ -264,7 +273,9 @@ \subsection{Wave propagation methods}\label{sec:waveprop}
 a detailed description of these two Riemann solvers in the context of
 the shallow water equations.
 Neither of these solvers deals with the carbuncle instability in a fully
-satisfactory way.  Rusanov's solver suppresses the carbuncle but is known to
+satisfactory way.  Rusanov's solver suppresses the carbuncle but 
+(due to the use of excessive numerical dissipation)
```

Comment:
I would like to keep this the way it was.  The new version makes it sound like the numerical dissipation is not inherent to Rusanov's solver.

Reply:
Ok, I see your point. But to me "Rusanov's solver suppresses the carbuncle but is known to also suppress related real instabilities" sounds misleading. When I read it, I understood as if that is what Rusanov's solver does, suppress real instabilities. This is not the case if the mesh is properly refined. Maybe we can say something like:

"Rusanov's solver suppresses the carbuncle but (unless the mesh is highly refined) is known to also suppress related real instabilities"

Reply:
Good point.  Your suggestion sounds good.  Or something like "Rusanov's solver suppresses the carbuncle instability but may introduce excessive numerical dissipation."  We don't have to talk about the physical instability here; it's enough to point out that there are drawbacks to using Rusanov in general.

Reply:
Yes, I like your last suggestion better. I did and pushed that change already.